x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/add_prepackaged_rules_schema.ts

```diff
@@ -79,5 +80,6 @@ export const addPrepackagedRulesSchema = Joi.object({
   type: type.required(),
   threat: threat.default([]),
   references: references.default([]),
+  note: note.default('').default(''),
```

Comment:
Default twice? Should it just be once here? `note.default('')`

Reply:
Below you're using:

```ts
note: note.allow('').default(''),
```

but here you have default twice. Maybe you wanted `allow('')` here?

Reply:
Yes! That was my mistake. However, I took your suggestion of not defaulting this value so not it's just ``` note: note.allow('') ```